### PR TITLE
Add test for get stored flows from flow_manager

### LINKED ISSUE #174: Add sdntrace_cp e2e test
This is to test sdntrace_cp when computing the trace from flow_manager stored_flows, as well as to check that [e2e sdntrace_cp test case](https://github.com/amlight/kytos-end-to-end-tests/blob/master/tests/test_e2e_40_sdntrace.py#L240-L241) doesn't fail anymore.

### PR DESCRIPTION
**Pull request by gretelliz**
Fix #174 

 Add `test_e2e_40_sdntrace.py::TestE2ESDNTrace::test_030_run_sdntrace_for_stored_flows` to test how the traces are obtained from flow_manager stored_flow.

Also, `test_e2e_40_sdntrace.py::TestE2ESDNTrace::test_001_run_sdntrace_cp` hass passed.

### CODE REVIEW

File: tests/test_e2e_40_sdntrace.py [hunk 1]
```diff
@@ -376,3 +376,23 @@ def test_020_run_sdntrace_fail_missing_flow(self):
             (step['dpid'], step['port']) for step in result[1:-1]
         ]
         assert expected == actual, f"Expected {expected}. Actual: {actual}"
+
+    def test_030_run_sdntrace_for_stored_flows(self):
```
Comment (viniarck) [inline tests/test_e2e_40_sdntrace.py:380]: This test case is relatively very similar to what `test_001_run_sdntrace_cp` covers. Maybe you could cover the `traces/` endpoint here instead to increase coverage? Covering with 2 or more traces the `traces/` endpoint would be great to have.

File: tests/test_e2e_40_sdntrace.py [hunk 2]
```diff
@@ -57,7 +57,7 @@ def get_evc(circuit_id):
 
 
     # This test may eventually fail due to https://github.com/kytos-ng/flow_stats/issues/19
-    @pytest.mark.xfail
+    # @pytest.mark.xfail
```
Comment (viniarck) [inline tests/test_e2e_40_sdntrace.py:60]: @gretelliz, great to know that we can remove the xfail mark. But, let's delete the line, including the comment of the previous line that is obsolete.

File: kytos-init.sh [hunk 3]
```diff
@@ -21,9 +21,9 @@ test -z "$TESTS" && TESTS=tests/
 test -z "$RERUNS" && RERUNS=2
 
 python3 scripts/wait_for_mongo.py 2>/dev/null
-python3 -m pytest $TESTS --reruns $RERUNS -r fEr
+#python3 -m pytest $TESTS --reruns $RERUNS -r fEr
```
Comment (viniarck) [inline kytos-init.sh:24]: This was accidentally committed

**Review by viniarck — CHANGES_REQUESTED** [review]
Gretel, 

The test case for the bulk endpoint it'd be great to cover more than one trace, since that's the typically how the bulk endpoint will be used, can you adapt the test case?

Also there is accidental committed changes.

**Comment by gretelliz** [comment]
@viniarck, note that now the `/traces` endpoint is tested for 4 traces, two of them are the same, so only 3 are returned. Two of them correspond to a same switch and the other to another switch.

**Review by viniarck — APPROVED** [review]
LGTM. It's great to have this coverage @gretelliz

**Pull request merged**